source/extensions/filters/http/gcp_authn/gcp_authn_filter.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+#include <memory>
+#include <string>
+
+#include "envoy/extensions/filters/http/gcp_authn/v3/gcp_authn.pb.h"
+#include "envoy/extensions/filters/http/gcp_authn/v3/gcp_authn.pb.validate.h"
+
+#include "source/common/http/message_impl.h"
+#include "source/extensions/filters/http/common/factory_base.h"
+#include "source/extensions/filters/http/common/pass_through_filter.h"
+#include "source/extensions/filters/http/gcp_authn/gcp_authn_impl.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace GcpAuthn {
+
+inline constexpr char FilterName[] = "envoy.filters.http.gcp_authn";
+inline constexpr char AudienceKey[] = "audience_key";
```

Comment:
prefer `absl::string_view` for better performance in map lookup. (without constructing std::string)

Reply:
Both string_view and char array require copies to convert to string (one implicit, one explicit conversation). But yes, string_view has better performance because string len is already computed.